### PR TITLE
New package: LazapDevelopment.Lazap version 0.6.5

### PR DESCRIPTION
**Pull request by voidedlegacy**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/112013&drop=dogfoodAlpha

### CODE REVIEW

**Comment by jboecker07** [comment]
@microsoft-github-policy-service agree [company="{Lazap Deveopment}"]

**Comment by jboecker07** [comment]
@microsoft-github-policy-service agree company="Lazap Development"

**Comment by jboecker07** [comment]
@microsoft-github-policy-service agree

File: manifests/l/LazapDevelopment/Lazap/0.6.5/LazapDevelopment.Lazap.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: LazapDevelopment.Lazap
+PackageVersion: 0.6.5
+PackageLocale: en-US
+Publisher: Lazap Development
```
Comment (Trenly) [inline manifests/l/LazapDevelopment/Lazap/0.6.5/LazapDevelopment.Lazap.locale.en-US.yaml:7]:
```suggestion
Publisher: lazap
```
This should match the value written to control panel

File: manifests/l/LazapDevelopment/Lazap/0.6.5/LazapDevelopment.Lazap.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: LazapDevelopment.Lazap
+PackageVersion: 0.6.5
+PackageLocale: en-US
+Publisher: Lazap Development
+PackageName: Lazap
+License: CC-BY-NC-SA-4.0
+ShortDescription: "Lightweight cross-platform client, unifying games into a single, modern library \U0001F3AE"
```
Comment (Trenly) [inline manifests/l/LazapDevelopment/Lazap/0.6.5/LazapDevelopment.Lazap.locale.en-US.yaml:10]:
```suggestion
ShortDescription: "Lightweight cross-platform client, unifying games into a single, modern library 🎮"
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by stephengillie — DISMISSED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**